certif/tests/unit/models/certification-candidate-test.js

```diff
@@ -189,6 +190,34 @@ module('Unit | Model | certification-candidate', function (hooks) {
     });
   });
 
+  module('#get accessibilityAdjustmentNeededLabel', function () {
+    test('should display "Yes" if adjusted certification is needed', function (assert) {
```

Comment:
**Proposition** : 

module ('when adjusted certification is needed' ///
   test('should display "yes"' ///
);
module ('when adjusted certification is not needed' ///
   test('should not display anything' ///
);